sdks/python/apache_beam/pipeline_test.py

```diff
@@ -280,9 +280,10 @@ def raise_exception(exn):
         # pylint: disable=expression-not-assigned
         p | Create([ValueError]) | Map(raise_exception)
 
-  def test_eager_pipeline(self):
-    p = Pipeline('EagerRunner')
-    self.assertEqual([1, 4, 9], p | Create([1, 2, 3]) | Map(lambda x: x*x))
+  # TODO(BEAM-1894).
```

Comment:
Could this be reproduced with existing custom sources? If that is the could we try to address the issue first instead of disabling tests?